manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorServiceImpl.java

```diff
@@ -304,34 +309,41 @@ public List<Processor> findByShardIdWithReadyDependencies(String shardId) {
     @Override
     public Processor updateProcessorStatus(ProcessorDTO processorDTO) {
         Bridge bridge = bridgesService.getBridge(processorDTO.getBridgeId());
-        Processor p = processorDAO.findById(processorDTO.getId());
-        if (p == null) {
+        Processor processor = processorDAO.findById(processorDTO.getId());
+        if (processor == null) {
             throw new ItemNotFoundException(String.format("Processor with id '%s' does not exist for Bridge '%s' for customer '%s'", bridge.getId(), bridge.getCustomerId(),
                     processorDTO.getCustomerId()));
         }
 
+        // If an exception happened; make sure to record it.
```

Comment:
Store any error information returned for the Operator.

Reply:
@manstis minor, but this could be moved after the `if (ManagedResourceStatus.DELETED == updateDTO.getStatus())` check